### PR TITLE
Improve unit test code in namespace.go

### PR DESCRIPTION
**Pull request by tariq1890**
Some minor code improvements in the unit tests.

### CODE REVIEW

File: modules/k8s/namespace.go [hunk 1]
```diff
@@ -32,21 +32,21 @@ func CreateNamespaceE(t *testing.T, options *KubectlOptions, namespaceName strin
 
 // GetNamespace will query the Kubernetes cluster targeted by the provided options for the requested namespace. This will
 // fail the test if there is an error in creating the namespace.
```
Comment (yorinasub17) [inline modules/k8s/namespace.go]: Can you update the docs to mention that this will also fail the test if the namespace doesn't exist?

Comment (tariq1890) [reply modules/k8s/namespace.go]: Sure!

Comment (tariq1890) [reply modules/k8s/namespace.go]: Done

**Comment by yorinasub17** [comment]
Build passed, so going to merge this. Thanks for your contribution!

**Pull request merged**